src/main/java/in/projecteka/consentmanager/hipLinkInitialization/Constants.java

```diff
@@ -0,0 +1,12 @@
+package in.projecteka.consentmanager.hipLinkInitialization;
```

Comment:
IMO, this should not be a top level package. 
I think it should be a package under consentmanager/link/

Reply:
done